package.json

```diff
@@ -69,6 +69,7 @@
     "@types/jest": "^27.5.1",
     "@types/node": "^18.11.18",
     "babel-loader": "^9.1.2",
+    "browserslist": "^4.21.5",
```

Comment:
I don't understand why browserslist needs to be added as a devDependency - `@shopify/browserslist-config` never had a transitive dependency on this, and you never call it, as I believe it gets pulled in via some babel dependency. I'm pretty sure this can be removed